### PR TITLE
feat(@clayui/css): LPD-42908 Adds language flags fr-BE, fr-CH, de-CH, it-CH, mk-MK, de-AT

### PR DESCRIPTION
**Pull request by pat270**
https://liferay.atlassian.net/browse/LPD-42908

### CODE REVIEW

File: clayui.com/static/js/icons-autogenerated.json [hunk 1]
```diff
@@ -679,6 +679,10 @@
         "name": "flag-full",
         "aliases": ["banner", "report"]
     },
+    {
+        "name": "du-be",
+        "aliases": [""]
+    },
```
Comment (matuzalemsteles) [inline clayui.com/static/js/icons-autogenerated.json:685]: @pat270 maybe this should be in flags or is this intentional?

Comment (pat270) [reply clayui.com/static/js/icons-autogenerated.json:685]: Not intentional, let me figure out why it was placed there.

Comment (pat270) [reply clayui.com/static/js/icons-autogenerated.json]: @matuzalemsteles should be good now. There was a documentation error in the issue `du-BE` isn't a language code. It should be `nl-BE` which is already in Clay.

**Review by matuzalemsteles — APPROVED** [review]
LGTM

File: clayui.com/static/js/icons-autogenerated.json (hunk 1 above)
Comment (matuzalemsteles) [reply clayui.com/static/js/icons-autogenerated.json]: Thanks @pat270

**Pull request merged**